spec/forms/waste_exemptions_engine/exemptions_form_spec.rb

```diff
@@ -12,7 +12,7 @@ module WasteExemptionsEngine
     end
 
     subject(:form) { build(:exemptions_form) }
-    let(:three_exemptions) { Exemption.order("RANDOM()").last(3) }
+    let(:three_exemptions) { create_list(:exemption, 3) }
```

Comment:
Now that we are using `create_list` here instead of Exemptions from the DB, is the comment in the `before` block above still accurate?

Reply:
Oh that's were the DB was populated from! 😂 missed it. I think the entire `before` block can go. Thanks for pointing that out :)